core/SConscript.firmware

```diff
@@ -15,7 +15,7 @@ PYOPT = ARGUMENTS.get('PYOPT', '1')
 DISABLE_OPTIGA = ARGUMENTS.get('DISABLE_OPTIGA', '0') == '1'
 HW_REVISION = ARGUMENTS.get('HW_REVISION', None)
 THP = ARGUMENTS.get('THP', '0') == '1' # Trezor-Host Protocol
-NEW_RENDERING = ARGUMENTS.get('NEW_RENDERING', '1') == '1' or TREZOR_MODEL in ('T3T1',)
+NEW_RENDERING = ARGUMENTS.get('NEW_RENDERING', '1') == '1'
```

Comment:
we should be dropping this as an argument, no? we want to go new rendering all the way for firmware, the only holdout is TT bootloader

Reply:
I intended this as a switch only, tried to not remove any old rendering related functionality for now. 

Obviously we should do clean-up and remove it all at some point, but i think its fine to postpone that a bit, so we still can switch for comparison when debugging etc.

Dtto for the comments below.